source/vir/src/def.rs

```diff
@@ -579,24 +582,36 @@ fn atomicity_type_name(atomicity: InvAtomicity) -> Ident {
 pub fn datatype_invariant_path(vstd_crate_name: &Option<Ident>, atomicity: InvAtomicity) -> Path {
     Arc::new(PathX {
         krate: vstd_crate_name.clone(),
-        segments: Arc::new(vec![
-            Arc::new("pervasive".to_string()),
-            Arc::new("invariant".to_string()),
-            atomicity_type_name(atomicity),
-        ]),
+        segments: Arc::new(if vstd_crate_name.is_some() {
+            vec![Arc::new("invariant".to_string()), atomicity_type_name(atomicity)]
+        } else {
+            vec![
+                Arc::new("pervasive".to_string()),
+                Arc::new("invariant".to_string()),
+                atomicity_type_name(atomicity),
+            ]
+        }),
```

Comment:
Is the plan to track these branches (for removal) by removing the `vstd_crate_name` bindings when we drop support for the old-style pervasive?

Reply:
Yes, the old code here will be removed when we drop the old-style pervasive.  Anything in `rust_verify` or `vir` that refers to pervasive would be removed.  We would not have to pass `vstd_crate_name` around everywhere and test for `vstd_crate_name.is_some()`.